app/modules/asset_groups/models.py

```diff
@@ -106,8 +125,6 @@ def setup(self):
             # We sometimes create an AGS when the percentage is 99%
             self.post_preparation_hook()
 
-        AuditLog.user_create_object(log, self)
```

Comment:
thinking this should stay?

Reply:
Agreed, I've added it back in where `ags.setup()` is called after creation